playground/fonts.css

```diff
@@ -0,0 +1,171 @@
+@font-face {
+    font-family: Proxima Nova;
+    font-style: normal;
+    font-weight: 400;
+    src: url(https://op3static.oktacdn.com/assets/loginpage/font/assets/proximanova-reg-webfont.9d5837512674046fa816.eot);
```

Comment:
Can you file a ticket and add a comment in this file to address this hard-coded preview static URL?